docs/docs/airtable/oauth-migration-2024-02/README.md

```diff
@@ -0,0 +1,78 @@
+# Update to the Airtable Integration on Pipedream (January 2024)
+
+Effective February 1st 2024, Airtable's API Key authentication method will be deprecated. To learn more about this change, please visit Airtable’s [dedicated support page](https://support.airtable.com/docs/airtable-api-key-deprecation-notice).
+
+### How will this impact my workflows?
+Starting February 1st 2024, all Pipedream steps using the legacy Airtable (API Key) integration including triggers and actions will no longer be able to authenticate with Airtable. 
+
+### What do I need to do?
+
+1. **Reconnect your Airtable account**: Visit [https://pipedream.com/accounts](https://pipedream.com/accounts) and search for Airtable and connect your account. This newer Pipedream integration uses OAuth instead of an API Key. 
+
+![Airtable Account Connection](https://res.cloudinary.com/dpenc2lit/image/upload/v1704326732/Screenshot_2024-01-03_at_4.02.24_PM_kvasnc.png)
+
+You can determine which workflows are connected to the legacy Airtable (API Key) app by expanding the account row.
```

Comment:
I'd maybe just clarify that users will need to expand the row for the API key account, not the new one they've connected in step 1.